### PR TITLE
URL structure to lowercase 

### PR DESCRIPTION
**Pull request by noobtuber20103152**
## Related Issue

issue #365 

url working correctly , please review and merge it @priyankarpal 

## Screenshots

image

image

image

image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -36,11 +36,11 @@ function App() {
...
           <Routes>
-            <Route path="/Projects/:username" element={<ProjectList />} />
+            <Route path="/projects/:username" element={<ProjectList />} />
             <Route path="/" element={<HomePage />} />
-            <Route path="/ProjectsPage" element={<ProjectsPage />} />
-            <Route path="/ContributorsPage" element={<ContriButorsPage />} />
-            <Route path="/AddYourProjectsGuide" element={<AddYourProjectsGuide />} />
+            <Route path="/projects-page" element={<ProjectsPage />} />
+            <Route path="/contributors-page" element={<ContriButorsPage />} />
+            <Route path="/add-your-projects-guide" element={<AddYourProjectsGuide />} />
```
Comment (priyankarpal) [inline src/App.jsx:43]: remove `-` & rename `add-your-projects-guide` to `docs`
``` suggestion
            <Route path="/projectspage" element={<ProjectsPage />} />
            <Route path="/contributorspage" element={<ContriButorsPage />} />
            <Route path="/docs" element={<AddYourProjectsGuide />} />
```

File: src/components/Navbar.jsx [hunk 2]
```diff
@@ -49,23 +49,23 @@ const Navbar = () => {
             </li>
             <li>
               <NavLink
-                to="/ProjectsPage"
+                to="/projects-page"
```
Comment (priyankarpal) [inline src/components/Navbar.jsx:52]:
``` suggestion
                to="/projectspage"
```

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -49,23 +49,23 @@ const Navbar = () => {
...
                 className="inline-block py-2 px-3 text-center font-bold rounded-md  focus:outline-none focus:ring focus:ring-gray-800 "
               >
                 Projects
               </NavLink>
             </li>
             <li>
               <NavLink
-                to="/AddYourProjectsGuide"
+                to="/add-your-projects-guide"
```
Comment (priyankarpal) [inline src/components/Navbar.jsx:60]:
``` suggestion
                to="/docs"
```

File: src/components/Navbar.jsx [hunk 4]
```diff
@@ -49,23 +49,23 @@ const Navbar = () => {
...
                 className="inline-block py-2 px-3 text-center font-bold  rounded-md   focus:outline-none focus:ring focus:ring-gray-800"
               >
                 Docs
               </NavLink>
             </li>
             <li>
               <NavLink
-                to="/ContributorsPage"
+                to="/contributors-page"
```
Comment (priyankarpal) [inline src/components/Navbar.jsx:68]:
``` suggestion
                to="/contributorspage"
```

**Comment by noobtuber20103152** [comment]
changes are done please verify @priyankarpal

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**